File: CS/DragDropFromGridExample/Model/Patient.cs
﻿using DevExpress.Mvvm.POCO;
using System;

namespace DragDropFromGridExample {
    public class Patient {
        public static Patient Create() {
            return ViewModelSource.Create(() => new Patient());
        }

        protected Patient() {
        }
        public virtual int Id { get; set; }
        public virtual string Name { get; set; }
        public virtual DateTime BirthDate { get; set; }
        public virtual string Phone { get; set; }
        public override string ToString() {
            return Name;
        }
    }
}

File: VB/DragDropFromGridExample/Model/Patient.vb
Imports DevExpress.Mvvm.POCO

Namespace DragDropFromGridExample

    Public Class Patient

        Public Shared Function Create() As Patient
            Return ViewModelSource.Create(Function() New Patient())
        End Function

        Protected Sub New()
        End Sub

        Public Overridable Property Id As Integer

        Public Overridable Property Name As String

        Public Overridable Property BirthDate As Date

        Public Overridable Property Phone As String

        Public Overrides Function ToString() As String
            Return Name
        End Function
    End Class
End Namespace
